doc/liquid/drops.html

```diff
@@ -1890,7 +1890,7 @@ <h2>
 <h3>
 <a id="name-14" class="anchor" href="#name-14" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>name</h3>
 <p>Returns name of your organization. That can be changed via the [admin dashboard][provider-account-edit].</p>
-<pre lang="liquid"><code>&lt;div&gt;Domain {{ provider.domain }}&lt;/div&gt;
+<pre lang="liquid"><code>&lt;div&gt;Domain {{ provider.external_domain }}&lt;/div&gt;
```

Comment:
See another comment, on `lib/developer_portal/lib/liquid/drops/provider.rb`, I guess this is generated from there.